docs/user/integrations/ipfabric.md

```diff
@@ -118,7 +118,7 @@ If you would like to change the default status change value, ensure you provide
 
 ![Safe Delete](../../images/ipfabric-safe-delete.png)
 
-An example object that's been modified by SSoT App and tagged as `ssot-safe-delete` and `ssot-synced-from-ipfabric`. Notice the Status and child object, IPAddress has also changed to Deprecated and, it's status changed and tagged as well.
+An example object that's been modified by SSoT App and tagged as `ssot-safe-delete` and `ssot_last_synchronized`. Notice the Status and child object, IPAddress has also changed to Deprecated and, it's status changed and tagged as well.
```

Comment:
Does `ssot-safe-delete` need to be changed too? What about updating the screenshot?

Reply:
I saw the `ssot-safe-delete` as a Tag not a CustomField so left it alone.

Reply:
Ah then maybe just a wording update and screenshot update? The screenshot shows both as tags?

Reply:
There is a Tag on the objects `SSoT Synced from IPFabric` and also a CustomField identifying IPFabric as the System of Record. The Tag is used for filtering what's imported depending upon the Job form option selection. I've updated the docs to clarify what's specifically being spoken about.